specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```diff
@@ -0,0 +1,1124 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Anomaly Detector Client",
+    "version": "1.1",
+    "description": "Multivariate Time-Series Anomaly Detection API"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "Ocp-Apim-Subscription-Key"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "schemes": [
+    "https"
+  ],
+  "paths": {
```

Comment:
Or possibly:
```
  "/multivariate/models/{modelId}/entire/detect": {       => /multivariate/models/{modelId}/:batch-detect
    "post": "BatchDetectAnomaly"
  }
```

Reply:
Should we do:
```
  "/multivariate/entire/detect/{resultId}": {                    => /multivariate/detect-batch/{resultId}
    "get": "GetDetectionResult"
  }
```

This would be fine.

Reply:
Hi Mike, what do you think we remove ":" in api path as below, this seems more friendly to our customers.

  "/multivariate/models/{modelId}/entire/detect": {       => /multivariate/models/{modelId}/detect-batch
    "post": "DetectAnomaly"
  }

  "/multivariate/models/{modelId}/last/detect": {          => /multivariate/models/{modelId}/detect-last
    "post": "LastDetectAnomaly"                                    => DetectLastAnomaly
  }

Reply:
REST API guidelines require that actions have `:` in the path. 
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action

Reply:
Fixed.